assets/css/edd-admin.css

```diff
@@ -1921,17 +1921,16 @@ body.download_page_edd-reports {
 }
 
 .edd-chip {
-	font-size: 7px;
+	font-size: 10px;
 	text-transform: uppercase;
 	font-weight: bold;
-	line-height: 8px;
+	line-height: 1;
 	vertical-align: middle;
 	padding: 2px 2px 3px 3px;
 	margin: -1px 0 0 2px;
```

Comment:
Looking at other uses of this component, I think we should remove the margin from the base and let anything implementing it to define its own margin.

The existing uses continue to look fine with the margin removed:

<img width="331" alt="Screen Shot 2020-04-14 at 9 25 59 AM" src="https://user-images.githubusercontent.com/491124/79230040-05439080-7e32-11ea-8257-f6e874d6577c.png">
<img width="228" alt="Screen Shot 2020-04-14 at 9 26 19 AM" src="https://user-images.githubusercontent.com/491124/79230044-05dc2700-7e32-11ea-9853-caa24781ebff.png">

But could be tweaked some in a follow up issue.